### PR TITLE
Add in new option for a radio group with more options accordion

### PR DESCRIPTION
**Pull request by robertparkinson**
https://tools.hmcts.net/jira/browse/ROC-2719

Link to prototype to see behaviour: 
https://hmcts-cmc-issue-prototype.herokuapp.com/prototype-dec-2017/certificate/upload

### CODE REVIEW

File: macros/form.njk [hunk 1]
```diff
@@ -123,7 +123,7 @@
 
 {% macro submitButton(label = 'Continue', id = '') %}
   <div class="form-group">
-    <input type="submit" class="button" value="{{ t(label) }}" {% if id %}id="{{ id }}"{% endif %}>
+    <input type="submit" class="button" value="{{ t(label) }}"{% if id %}name="{{ id }}"{% endif %} {% if id %}id="{{ id }}"{% endif %}>
```
Comment (timja) [inline macros/form.njk:126]: why is this needed?

**Review by timja — COMMENTED** [review]
could you add a couple of screenshots to show what these macros provide?

File: macros/form.njk (hunk 1 above)
Comment (robertparkinson) [reply macros/form.njk:126]: I can't remember now as it was before xmas.. I'll remove it and see what happens

Comment (robertparkinson) [reply macros/form.njk:126]: have added a link to the prototype so you can see what the end result looks like

Comment (robertparkinson) [reply macros/form.njk:126]: Have remembered. Need the name on the button so that when it posts back it is part of the form object so I can check which button has been clicked

**Review by timja — APPROVED** [review]
